packages/core/platform/bundles/paas-proxmox.yaml

```
@@ -0,0 +1,138 @@
+
+- name: redis-operator
+  releaseName: redis-operator
+  chart: cozy-redis-operator
+  namespace: cozy-redis-operator
+  dependsOn: [cilium,kubeovn]
+
+- name: proxmox-csi-operator
+  releaseName: proxmox-csi-operator
+  chart: cozy-proxmox-csi-operator
+  namespace: cozy-proxmox
+  dependsOn: [cilium,kubeovn,cert-manager]
```

yamllint

[warning] too few spaces after comma

(commas)